k2/csrc/array_ops.cu

```diff
@@ -144,16 +251,18 @@ void ExclusiveSumDeref(Array1<T *> &src, Array1<T> *dest) {
   PtrPtr src_data = PtrPtr(src.Data());
   T *dest_data = dest->Data();
 
-  // use the ExclusiveSum() template for pointer-like objects that is declared
+  // use the ExclusivePrefixSum() template for pointer-like objects that is declared
   // in utils.h
-  ExclusiveSum(src.Context(), dest_dim, src_data, dest_data);
+  // @todo: need implementations in utils.cu for <PtrPtr, T *> ?
```

Comment:
For `TODO`, should we use the style you use here (i.e. `@todo`), we have used `// TODO(name)` for a long time, but I'm not sure if this is more formal in doxygen. Just to make sure we 'll follow the same convention, it may help us to search TODO in the code.

Reply:
Lets still use TODO... I dont know if I want to totally buy into this doxygen stuff, I am anticipating that mostly people will be looking at the code itself.

Reply:
I suggest `@todo abcdefg` (without `:`). It's more doxygen. But don't take time to correct those. As both are searchable, and the whole comment style would be handled later.